journeyStar_shacl.ttl

```diff
@@ -145,7 +145,40 @@ js-shacl:JourneyShape a sh:NodeShape ;
     ] ,
     [
         sh:path js:transitThrough ;
-        sh:node js-shacl:LocationShape ;
+        sh:or ( [ sh:datatype xsd:string ]
```

Comment:
I noticed that locations are inconsistent. Was this on purpose that some properties have location targets that were more restrictive? I now changed all to the less restrictive option.

Reply:
It does not need to be strictly of type Location, making it generic like you have done is fine.